### DISCUSSION TITLE
Download URLs of Coqui models

### DISCUSSION THREAD

**Discussion by ROBERT-MCDOWELL** [idiap/coqui-ai-TTS#212] [Q&A] [answered]
when we use TTS('tts_models/multilingual/multi-dataset/xtts_v2') does it mean it download from HF official coqui-tts repo or else?

**Comment by eginhard** [comment, marked as answer]
Yes, you can check the source URLs for each model here: https://github.com/idiap/coqui-ai-TTS/blob/dev/TTS/.models.json

**Comment by ROBERT-MCDOWELL** [comment]
thanks @eginhard
